### PR TITLE
Perzistent filters

### PR DESCRIPTION
**Pull request by krtko1**
depends on #98, implements #103

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -69,11 +69,14 @@ const CONSTANTS = {
 
 const DATA_URL = 'https://raw.githubusercontent.com/kockatykalendar/data/gh-pages/2020_21.json'
 let DATA = []
-let FILTER = {
-	school: [0, 14],
-	sciences: ['mat', 'fyz', 'inf', 'other'],
-	organizers: ['trojsten', 'p-mat', 'sezam', 'strom', 'riesky', '*'],
-	default_organizers: ['trojsten', 'p-mat', 'sezam', 'strom', 'riesky'],
+let FILTER = JSON.parse(localStorage.getItem('filter'));
```
Comment (Zajozor) [inline app.js:72]: `let FILTER = JSON.parse(..) || { school: ... }`

File: app.js [hunk 2]
```diff
@@ -509,11 +514,24 @@ document.querySelectorAll('.double-slider').forEach(parent => {
...
+
 		clearTimeout(slider_timer);
 		slider_timer = setTimeout(() => {
 			render();
 			CALENDAR.refresh();
 		}, 300);
 	}, false);
 });
+
+document.querySelectorAll('.va').forEach( el => {
```
Comment (Zajozor) [inline app.js:527]: do these lines have something to do with persistent filters?

Comment (krtko1) [reply app.js:527]: yes, set double slider

File: app.js (hunk 1 above)
Comment (krtko1) [reply app.js]: fixed

**Comment by Zajozor** [comment]
The parts for persistent filter seem fine. did not view other commits.

File: app.js (hunk 2 above)
Comment (Zajozor) [reply app.js:524]: seems more like a double slider than persistent filter thing to me :D

**Review by gardenerik — APPROVED** [review]

**Pull request merged**